week_2_workflow_orchestration/README.md

```diff
@@ -81,19 +81,13 @@
 * [Prefect Cloud](https://app.prefect.cloud/)
 * [Prefect Slack](https://prefect-community.slack.com)
 
-### Homework 
-
-To be posted on Monday by 14:00 CET
+### Code repository
 
+[Code from videos](https://github.com/discdiver/prefect-zoomcamp) (with a few minor enhancements)
 
-### Extra Practice
+### Homework 
```

Comment:
We will need to create a separate file homework.md with more detailed instructions. But for now i guess it's fine

Reply:
Oh right. You removed the content. Never mind then =)